aws_key.pem

```diff
@@ -1,23 +0,0 @@
------BEGIN RSA PRIVATE KEY-----
-MIIEogIBAAKCAQEAm1YqTU8dnyIGrLrypvQTJ3zBmax5sWcECgPa6QjLez51Yl789dwuCwhb2b4G
-09IVt0diMibtFNFLr/d10+PXzjM8K7epqRZQVWMPL75Lv0n9wEPNL7ey1Wjz0jnG3eGKAwHQZbrY
-TE5+Rvqvj20Hhnu1Uph/VqKfXeyYfcOUtrA070n6hwxWw9+rZyp7hZPQVl6zFqhMOijOngnEJZxp
-5KcMGCSIAY6eSW6oo2EUZlmTOLQ5Tk6rcrDHvth4Q3SNOUNxonk3PpERxz4JG17VhhabXZqbv4L2
-6kL5gMFE9FsmN4WBYfOtrlHqxwHD172XrzLFDE/x4N37wa8zULr6lQIDAQABAoIBAEYVtVZWKNZA
-ouRSIOMk4nOBfVQZlnptMUxXevCDcFN11b6Bfv5qzjdaGR7/9tKhSfz8bxiTqYon+E83hmkItrip
-My/CWQEaOTqdTB/Nax2ug5e4v54U8tII3WFvggNgHEXJDr6nc6nSDHV367f5iFir/h57AZEUJHVb
-J14u3S0jVU0NlI2O2dwQnutRFrm7ODu9oj5SExdL9mYpKl9kwYIz/T31HrJV0TidmQu4qq9KSbcl
-sHXcSCxUoYHY+N7NEjERQdNLvdjPOEeCQllj82ie3ieHG9QOMRCQS/Fq/56VuRVrEUK2X1m87YrW
-xe+bRaZcE691rbyptoNLZ8FXKy0CgYEA0KykFEFl8yyLy23sUITjR9qwFveK5KUNY1RAfwF4eVNN
-a7FyWGoryuwnD7AF9tVcHhvAx4o39dG946xM67Q9FDFsnPCy/WcZnj1k5Cy4D+dsc8rdvXE26zF0
-970jSeLSuXuF29Yc4ZBm/UheNzLmUbFixrDTwQvbY9e/lRgT6v8CgYEAvpDNYdEEs9uHahc3Pc8O
-LOBNnhrEFFOeo1iK8EX9rp9PiVZj8vmIu6NzuFlxCIdLLYBYwHFJ3h2UDAp9QfL/U9pqu0VFWseN
-mwvFGB8xUrfh18wt72CgEWCj21OLIe/DzV7dw12GAMNzZ9Aa8PUjsV4dV1qiLdnR5qE0B+iCPmsC
-gYBHx0q9SO2XovgHJq2F2OARJFEz5Txaf8fGJgH1bf0Rxqy34ZbSgNrFd0vMYqD6kdOY67T5Ljsn
-jFkECo1++kWUP5f7t/PppGA0hSvCLPu3RMSSY/K4KkxYxG9WlDGiRap0cN/PiavvjutOLmlyFUkS
-SrEOZV7BZaG2i0z0LQ8U4wKBgGXPLLrWlYGVhkoKryupZDRbfXw7ZohJiFZSZqApdHeHi1GfQvS7
-le5RS/CwcDATP/nGOSRPIhy1hrPMItFKBlkBNsZoIr52IFQ2PI0/lPVRYioNXnkmET5XeBMkFhAC
-bYRZ63aFj6L2yMAssUgMX45R7rENrdl7JqXld8vFz1eTAoGAVTUYWXG84Qfjf0avZknlHF4/DoRv
-B4zYgab6obLNDyaJDTKCBORyl34ln3qDc4Av0AhYIR7VJwKVla5+b8GzqwXVCLY0UItu0uRdQI0p
-48jVO97MtKw+fHB8Z3J8jE+IjYpYN8maTkeqvXf/+8HJ/3BWVizZrDluvpoq+UNjBiU=
------END RSA PRIVATE KEY-----
```

Comment:
I don't know how I didn't see it on time. The only good thing is that the uploaded key was not working anyway, and it was removed afterwards.